unic/ucd/tests/category_tests.rs

```diff
@@ -18,8 +18,112 @@ extern crate unic_utils;
 
 use unic_ucd::bidi::BidiClass;
 use unic_ucd::normal::is_combining_mark;
+use unic_ucd::category::GeneralCategory as GC;
```

Comment:
Feel free to also `use ...::BidiClass as BC`, if you think makes the lines shorter and removes the clutter.